packages/manager/apps/zimbra/src/pages/dashboard/Domains/AddDomain.tsx

```diff
@@ -4,74 +4,209 @@ import { useTranslation } from 'react-i18next';
 import { useNavigate } from 'react-router-dom';
 import {
   OsdsFormField,
+  OsdsRadio,
+  OsdsRadioButton,
+  OsdsRadioGroup,
   OsdsSelect,
   OsdsSelectOption,
   OsdsSpinner,
   OsdsText,
+  OsdsButton,
+  OsdsInput,
 } from '@ovhcloud/ods-components/react';
 import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
 import {
+  ODS_INPUT_TYPE,
+  ODS_RADIO_BUTTON_SIZE,
   ODS_SPINNER_SIZE,
   ODS_TEXT_LEVEL,
   ODS_TEXT_SIZE,
+  OdsInputValueChangeEvent,
+  OdsSelectValueChangeEvent,
+  OsdsRadioGroupCustomEvent,
 } from '@ovhcloud/ods-components';
-import { useOrganization, useOrganizationList } from '@/hooks';
+import { useQuery } from '@tanstack/react-query';
+
+import { useOrganization, useOrganizationList, usePlatform } from '@/hooks';
+import {
+  getDomainsZoneList,
+  getDomainsZoneListQueryKey,
+  postZimbraDomain,
+} from '@/api';
 
 export default function AddDomain() {
-  const { t } = useTranslation('domains');
+  const { t } = useTranslation('domains/addDomain');
+  const { platformId } = usePlatform();
+
   const { data, isLoading } = useOrganizationList();
   const navigate = useNavigate();
   const { data: organization } = useOrganization();
   const [selectedOrganization, setSelectedOrganization] = useState(
     organization?.id || '',
   );
 
-  const handleSelectChange = (event: any) => {
-    setSelectedOrganization(event.detail.value);
+  const { data: domains } = useQuery({
+    queryKey: getDomainsZoneListQueryKey,
+    queryFn: () => getDomainsZoneList(),
+  });
+
+  const [selectedRadioDomain, setSelectedRadioDomain] = useState('');
+  const [selectedDomainName, setSelectedDomainName] = useState('');
+  const handleRadioDomainChange = (
+    event: OsdsRadioGroupCustomEvent<{
+      newValue?: string;
+      previousValue?: string;
+    }>,
+  ) => {
+    setSelectedRadioDomain(event.detail.newValue);
+  };
+
+  const handleInputChange = (event: OdsInputValueChangeEvent) => {
+    setSelectedDomainName(`${event.detail.value}`);
+  };
+
+  const handleSubmit = () => {
+    const formData = {
+      organizationId: organization?.id,
+      name: selectedDomainName,
+    };
+    postZimbraDomain(platformId, formData);
   };
   return (
-    <div className="flex flex-col items-start space-y-4">
+    <div className="flex flex-col items-start w-full md:w-3/4 space-y-4">
       <Links
         type={LinkType.back}
         onClickReturn={() => navigate('..')}
         label={t('zimbra_domains_add_domain_cta_back')}
       />
-      <Subtitle>{t('zimbra_domains_add_domain_title')}</Subtitle>
+
+      <div className="">
+        <Subtitle>{t('zimbra_domains_add_domain_title')}</Subtitle>
+      </div>
```

Comment:
```suggestion
        <Subtitle>{t('zimbra_domains_add_domain_title')}</Subtitle>
```
i'm not sure that is needed